config/settings/development.yml

```diff
@@ -18,7 +18,7 @@ action_kit:
 ak_api_url: 'http://actionkit_api_url.com'
 # ak_username: username
 # ak_password: password
-
+default_currency: 'USD'
```

Comment:
I added `default_currency` into `Settings` instead of hard coding 'USD' everywhere in the codebase. What this is used for is converting the amounts in the transactions into this default currency for persistence in the `total_donations` column in the `Pages` table. We won't need to add another environment variable for this because on production it checks if the env var is defined, and if not, it falls back to USD.